tests/network/vmi_multus.go

```diff
@@ -476,7 +480,35 @@ var _ = SIGDescribe("[Serial]Multus", Serial, decorators.Multus, func() {
 				Expect(strings.Contains(out, customMacAddress)).To(BeFalse())
 
 				By("Ping from the VM with the custom MAC to the other VM.")
-				libwait.WaitUntilVMIReady(vmiTwo, console.LoginToFedora)
+				Expect(libnet.PingFromVMConsole(vmiOne, ptpSubnetIP2)).To(Succeed())
+			})
+
+			It("vmi with an hotplugged interface has connectivity over the secondary network", func() {
+				By("Creating another VM with only the masquerade interface")
+				vmiOne := libvmi.NewFedora(
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				)
+				vmiOne = tests.CreateVmiOnNode(vmiOne, nodes.Items[0].Name)
+				vmiOne = libwait.WaitUntilVMIReady(vmiOne, console.LoginToFedora)
+				Eventually(matcher.ThisVMI(vmiOne), 12*time.Minute, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceAgentConnected))
```

Comment:
Why do you need to wait for the ga?

Reply:
I just copied this from the previous test; I think I can remove it.